modules/HapticGlove_module/src/RobotController.cpp

```diff
@@ -62,10 +62,10 @@ bool RobotController::configure(const yarp::os::Searchable& config,
 
     if (m_axesJointsCoupled)
     {
-        m_A.resize(m_numActuatedJoints, m_numActuatedAxis);
-        m_Bias.resize(m_numActuatedJoints, 1);
+        m_A = Eigen::MatrixXd::Identity(m_numActuatedAxis, m_numActuatedAxis);
+        m_Bias = Eigen::MatrixXd::Zero(m_numActuatedJoints, 1);
 
-        if (!m_doCalibration)
+        if (false && !m_doCalibration)
```

Comment:
Probably this ``false`` can be removed. @RiccardoGrieco @lrapetti, do you remember why it was needed?